test/bats/ln-receive.bats

```diff
@@ -69,9 +111,13 @@ usd_amount=50
 
   # Check for subscriber event
   check_for_ln_update "$payment_hash" || exit 1
+
+  exec_graphql "$token_name" 'account-details'
+  account_id="$(graphql_output '.data.me.defaultAccount.id')"
+  cat .e2e-callback.log | grep "$account_id" || exit 1
```

Comment:
`account_id` needs to be brought back into this test. Can be done by caching in 1st test and reading back here

```suggestion
  cat .e2e-callback.log | grep "$(read_value $token_name.account_id) || exit 1
```